src/server/middleware/https.js

```diff
@@ -0,0 +1,11 @@
+const shouldRedirect = (req) =>
+  !req.secure &&
+  req.get("x-forwarded-proto") !== "https" &&
+  process.env.NODE_ENV !== "development";
```

Comment:
I'm not sure if this should look for production instead of "not development."

```suggestion
   process.env.NODE_ENV === "production";
```